packages/utils/src/walk-object.ts

```diff
@@ -1,4 +1,4 @@
-import { isArray, isObject } from "@chakra-ui/utils"
+import { isArray, isObject } from "./assertion"
```

Comment:
`preconstruct` doesn't like packages importing from themselves without having themselves as dependencies 😆 

That was an unnecessary import anyway, so fixed it here.

Reply:
Could you report this as a bug to preconstruct? I think this works now in node by default, according to the resolution algorithm: https://nodejs.org/api/modules.html#modules_all_together , It looks like `LOAD_PACKAGE_SELF` is allowing that explicitly.

I recall having some problems with that in the past and the previous version of the algorithm did not have this explicit step. However, I can't see why `LOAD_NODE_MODULES` would not allow this in the old algorithm so maybe this has always worked?

Either way - I think it makes sense to support this in preconstruct so we should fix this.